tuf/utils/x509_test.go

```diff
@@ -239,6 +279,296 @@ func TestECDSAX509PublickeyID(t *testing.T) {
 	require.Equal(t, tufPrivKey.ID(), tufID)
 }
 
+func preserveEnv(name string) func() {
+	if env, has := os.LookupEnv(name); has {
+		os.Unsetenv(name)
+		return func() {
+			os.Setenv(name, env)
+		}
+	}
+
+	return func() {}
+}
+
+func TestExtractPrivateKeyAttributes(t *testing.T) {
+	if os.Getenv(t.Name()) != "1" {
```

Comment:
We should keep checks for the env variable consistent across tests and code.  I think the code [checks for any non-empty value](https://github.com/docker/notary/pull/1130/files#diff-d0c91ba31f94b1f64375d2c05e63424fR12) of `GOFIPS` so can we mimic the check here?